src/accordionPanelGroup.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import AccordionPanelGroupContainer from './AccordionPanelGroupContainer';
+
+describe('AccordionPanelGroupContainer', function() {
+
+	it('exists', function() {
+		const accordionPanelGroup = TestUtils.renderIntoDocument(<AccordionPanelGroupContainer />);
+		const accordionPanelGroupNode = ReactDOM.findDOMNode(accordionPanelGroup);
+
+		expect(accordionPanelGroupNode).not.toBeNull();
+	});
+
+});
```

Comment:
You check the coverage report when you run `yarn test` to make sure that all the new components have 100% test coverage.